src/applications/vaos/components/InfoAlert.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function InfoAlert({
```

Comment:
Might be good to do some JSDoc comments. We didn't really resolve the issue, but seemed like there was some value in both those and prop types.